scripts/import-candid

```diff
@@ -50,7 +50,9 @@ test -n "${IC_DIR:-}" || {
   echo "       Use --help for usage."
   exit 1
 } >&2
-IC_COMMIT="$(cd "${IC_DIR}" && git rev-parse HEAD)"
+IC_COMMIT="$(cd "${IC_DIR}" && git rev-parse --short HEAD)"
+IC_DATE="$(cd "${IC_DIR}" && git show -s --format=%as)"
+IC_TAGS="$(cd "${IC_DIR}" && git tag --points-at HEAD | tr "\n" " ")"
```

Comment:
Note: I did have a shot at getting all the above in one go, with e.g.:
```
$ git show --pretty=format:"%h (%as %(describe:tags))"
ec140b74dc (2023-11-08 release-2023-11-08_23-01)
```
But, as you see, describe:tags adds a suffix to the end of the tag and provides a string even if there are no tags.  I don't think that would be very helpful.